scripts/globals/gear_sets.lua

```diff
@@ -178,7 +181,9 @@ local gearSets =
              {id = 214, items = {26673, 26849, 27025, 27201, 27377}, matches = 2, matchType = matchtype.any, mods = {{xi.mod.DOUBLE_ATTACK, 4, 2, 0}} }, -- Argosy +1
```

Comment:
Can we find a way to incorporate the items.lua enum into this table?  It might mean expanding out the single-line format, but I think it would improve readability by a huge amount.

Reply:
If it becomes a requirement, I'll consider it as an option as I continue iterating through the original PR to get to a finished state in future PRs. Though, tbh, I'm not sure it's a good fit here. Opinion follows:

I would point out that moving to ENUMs instead of Item IDs would likely add an additional layer of complexity to the process since we'd have to not only enumerate all the item IDs, but then migrate them into `items.lua` and additionally spot check for typos and item name inconsistencies across both files (e.g. would `27442` be `HASHISHIN_BASMAK_+1` or `HASHI._BASMAK_+1`?).

In a finished state, this might make this a bit more human readable, but any gains seem like they'd be outweighed by the time spent doing the conversion and the added complexity. My goal is to have the comments on each set line give enough additional contextual information to be used in conjunction with a cursory SQL look up to confirm the items:

```sql
SELECT * FROM xidb.item_basic
WHERE itemid in (26770, 26771, 26928, 26929, 27082, 27083, 27267, 27268, 27441, 27442);
```
`-- AF3 BLU 109/119 Set: Occasionally triples the WSC of Blue Magic Spells. Will stack with Chain Affinity.`

In a "finished" state, we'd see each line commented, with an easily selected array of item ids that can be plugged into SQL if there was any need to get more info.

Thoughts?

Reply:
my 2 cents: I think while dealing with item names won't be that different from dealing with item ids (they get typo'd too) I think there's already enough to chew on in the incoming work here, and putting it to names can be done as its own thing at a later date if it really needs doing (I may even volunteer myself for it, I am good with that sort of drudgery when my clients at work take 20 min to figure out how to unmute their mic).

Reply:
p.s. we've been using the convention `_P1` and `_p2` to mean `+1` and `+2` and stripping our symbols like periods, in the existing enum file, just so you know about that.

Reply:
Any additional feedback on the PR? Does the dialog regarding itemid vs utilizing items.lua prevent this from moving forward in it's current state?

Reply:
@claywar I've got no further input for the man, You?

Reply:
Only other thing I would suggest is removing the debug prints if they're no longer needed